docs/versions.md

```diff
@@ -38,21 +38,50 @@ most target .NET Standard 1.1.
 [1.6]: versions/netstandard1.6.md
 [2.0]: versions/netstandard2.0.md
 
-**NOTE**: The table above reflects the mappings that will happen when we release the
-.NET Standard 2.0 tooling. You can see that .NET Framework 4.6.1 mapping is being moved from
-1.4 to 2.0.
-
 ## How do I know which .NET Standard version I should target?
 
 When choosing a .NET Standard version, you should consider this trade-off:
 
 * The higher the version, the more APIs are available to you.
 * The lower the version, the more platforms implement it.
 
-So generally speaking, you should target the lowest version you get away with.
+So generally speaking, consider the following guidelines:
+
+* You should target the lowest version you get away with.
```

Comment:
Should be "You should target the lowest version you can get away with."

Reply:
Fixed